src/state-manager/TransactionQueue.ts

```diff
@@ -718,59 +718,59 @@ class TransactionQueue {
       spreadTxToGroupSyncingBinaryHandler.handler
     )
 
-    this.p2p.registerGossipHandler(
-      'spread_tx_to_group',
```

Comment:
This needs to remain

Reply:
thura corrected the fix and i also couldn't anything else which was still getting used .. with the commented endpoints in the pr.. @mssabr01